rollup.config.js

```diff
@@ -15,6 +16,10 @@ const commonPlugins = [
     }),
     sourceMaps(),
     terser(),
```

Comment:
Can you comment out terser? Separate from your work, I wanna see if this is impacting intellisense.